SU-HSDO/suhumsci
# Hsd8-653 Generic 3 column 1-5-1 variant

# READY FOR REVIEW

# Summary
- Added variant to 3 column pattern for 1 - 5 - 1 organization. This allows the middle column to always be 5 times wider than the sides.

# Needed By (Date)
- ASAP

# Urgency
- Medium. Math site needs it.

# Steps to Test
1. Checkout this branch
1. clear cache
1. view `/patterns/generic_three_column` page and ensure the "Small Side Columns" variant has a middle are that is 5 times wider than the sides

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
